geode-protobuf/src/main/java/org/apache/geode/internal/serialization/registry/exception/CodecAlreadyRegisteredForTypeException.java

```diff
@@ -21,7 +21,7 @@
  * handler for.
  */
 @Experimental
-public class CodecAlreadyRegisteredForTypeException extends Exception {
+public class CodecAlreadyRegisteredForTypeException extends RuntimeException {
```

Comment:
I would keep this as a checked exception as this is a condition that the system should realistically be able to recover from.